sbt_common/internal_model/src/test/scala/uk/ac/wellcome/models/work/test/util/WorksGenerators.scala

```diff
@@ -160,4 +163,55 @@ trait WorksGenerators extends ItemsGenerators {
     (1 to count).map { _ =>
       createIdentifiedWork
     }
+
+  def createSierraWorkWith(
```

Comment:
nit: could this be `createUnidentifiedSierraWorkWith`?

We had some `createWork`-like method names in the past, and it got horribly confusing trying to work out what was being created!

Reply:
good point.